rcon/player_history.py

```diff
@@ -352,10 +352,22 @@ def save_start_player_session(
             )
             return
 
+        start_time = datetime.datetime.fromtimestamp(timestamp)
+        already_saved = (
+            sess.query(PlayerSession)
+            .filter(PlayerSession.steamid == player)
+            .filter(PlayerSession.start == start_time)
+            .first()
```

Comment:
first() doesn't raise if there's no result? I tend to use `one_or_none()`

Reply:
Docs say it returns none:
https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.first
> Return the first result of this Query or None if the result doesn’t contain any row.